sources/osgViewer/webgl-utils.js

```diff
@@ -110,7 +115,7 @@ var WebGLUtils = function () {
         /** function:(msg) */
         opt_onError ) {
         function handleCreationError( msg ) {
-            if ( msg.indexOf( 'WebGL2' ) !== -1 ) return;
+            if ( !doReportCreationError ) return;
```

Comment:
It will not work because setupWebGL call create3DContext and create3DContext set the global variable doReportCreationError to false, the message will never be displayed, unless I missed something.
Maybe just add a optional parameter to function instead of global variable